CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# dbt_salesforce_formula_utils v0.10.0
+[PR #109](https://github.com/fivetran/dbt_salesforce_formula_utils/pull/109) includes the following updates:
+
+## 🚨 Breaking Changes 🚨
+- As announced in the [v0.9.2 August 2023 release](https://github.com/fivetran/dbt_salesforce_formula_utils/releases/tag/v0.9.2), the parameter `full_statement_version=false` has now been fully sunset from the `sfdc_formula_view` macro. You will now need to remove this parameter to avoid a compilation error.
+  - As part of this deprecation, the parameters `reserved_table_name` and `fields_to_include` have also been deprecated and will also need to be removed. 
```

Comment:
Can you also include a sub bullet mentioning all of the macros that were removed as part of this update.

Reply:
Good call. Updated!